src/components/MCQ/MCQButton.vue

```diff
@@ -15,22 +14,39 @@
 import { ref } from "vue";
 import type { MCQButton } from "@type/MCQ.d.ts";
 
-const { submitted, buttonDisabled } = defineProps<MCQButton>();
-const buttonClass = ref<string>("submit");
-const buttonText = ref<string>("Submit");
-const emit = defineEmits(["submitAnswer", "nextQuestion"]);
+const { submitted, selectedOption } = defineProps<MCQButton>();
+const buttonClass = ref<string>("skip");
+const buttonText = ref<string>("Skip");
+const emit = defineEmits(["submitAnswer", "nextQuestion", "skipQuestion"]);
 
-const handleButtonClick = (submittedValue: boolean) => {
-  if (!submittedValue) {
+const handleButtonClick = (
+  submittedValue: boolean,
+  selectedOptionValue: string | null,
+) => {
+  if (!submittedValue && selectedOptionValue) {
     emit("submitAnswer");
     buttonClass.value = "next";
     buttonText.value = "Next";
-  } else if (submittedValue) {
+  } else if (submittedValue && selectedOptionValue) {
     emit("nextQuestion");
-    buttonClass.value = "submit";
-    buttonText.value = "Submit";
+    buttonClass.value = "skip";
+    buttonText.value = "Skip";
+  } else if (!submittedValue && !selectedOptionValue) {
+    emit("skipQuestion");
+    buttonClass.value = "skip";
+    buttonText.value = "Skip";
   }
```

Comment:
This looks repetitive :) :) :)

Wrap in a fn?: `modifyButtonAndEmit(class, text, event)`